client/snap_op.go

```diff
@@ -204,9 +204,7 @@ func (client *Client) doMultiSnapActionFull(actionName string, snaps []string, o
 		Snaps:  snaps,
 	}
 	if options != nil {
-		if options.Dangerous {
-			return nil, "", ErrDangerousNotApplicable
-		}
+		// TODO consider returning error when options.Dangerous is set
```

Comment:
thanks for adding this

nitpick, we tend to format these as: // TODO: ...

with a colon (:)